### DISCUSSION TITLE
Using model inside query

### DISCUSSION THREAD

**Discussion by allestaire** [prisma/prisma#12983] [Q&A] [answered]
I notices that, when using model inside query, the computed fields did not appear.
Is this something the limitation or theres should be any extra steps ?
What the code does is
Query.js
```javascript
const getItem ( ... ) => {
    ...
}

const getHowManyItems ( ... ) => {
   ...
   // Reusing query
   getItem( ... )
   // Item has computed fields but did not appear
   return ...
}
```

What I think is, computed fields only work when the query call is done on resolver

**Comment by pantharshit00** [comment]
Hello @allestaire , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

If this is still relevant, can you please share `getItem` function? It is hard to infer otherwise what exactly that function is doing.

**Reply by vladi-stevanovic** [reply]
👋 Hello @allestaire I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by allestaire** [comment, marked as answer]
Hi, thanks for notifying,
Already fix the issue but somehow I dont know how I did it.

Cheers,

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
